static/elements/chromedash-color-status.mjs

```diff
@@ -0,0 +1,34 @@
+import { LitElement, html } from 'lit-element';
+
+class ChromedashColorStatus extends LitElement {
+  static get properties() {
+    return {
+      max: {
+        type: Number,
+      },
+      value: {
+        type: Number,
+      }
+    };
+  }
+
+  constructor() {
+    super();
+    this.max = 7;
+  }
+
+  render() {
+    return html`
+      <link rel="stylesheet" href="/static/css/elements/chromedash-color-status.css">
+      <span id="status" style="background-color: ${this._getColor()}"></span>
+    `;
+  }
+
+  _getColor() {
```

Comment:
a couple of options here, though this is fine as-is:

1. If `_getColor` is only used in `render()` you could just compute the value directly in `render()`.
2. If this were a very high-frequency, performance sensitive element, and you wanted to avoid calculating this derived value every time, you could memoize it in `update()`:

```js
update(changedProps) {
  if (changedProps.has('value')) {
    this._color = ....
  }
  super.update(changedProps);
}
```

or use the `guard()` directive:

```js
import {guard} from 'lit-html/directives/guard.js';
//...
render() {
  return html`
    <span id="status" style="background-color: ${guard([this.value], () => this._getColor()}"></span>
  `;
}
```

Of course, this isn't an expensive operation here, but I just wanted to point this out in case there is one in the future.